<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instellingen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/icons.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Nieuwsgroepen server</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server adres</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="serverEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gebruikersnaam</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Wachtwoord</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="passEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="sslBox">
        <property name="text">
         <string>SSL encryptie</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="xzverBox">
        <property name="text">
         <string>Header compressie (Astraweb/XZVER/XZHDR)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="nntpSwitchCheck">
        <property name="text">
         <string>Provider met  aparte backends voor txt/binary (NNTPswitch)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Poort</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="portEdit">
        <property name="text">
         <string>119</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Spots</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Max. aantal headers binnenhalen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="maxSpots">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10000000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000000000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Max. aantal dagen bewaren</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="maxDays">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>7</number>
        </property>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>365</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>900</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>99999</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Je schuilnaam (wordt vermeld bij reacties)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="nickEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Anoniempje</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="imagesCheck">
        <property name="text">
         <string>Plaatjes van externe websites tonen</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="nntpImagesCheck">
        <property name="text">
         <string>Plaatjes uit nieuwsgroepen tonen</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="eroCheck">
        <property name="text">
         <string>Erotiek tonen</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="trashCheck">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Prullenbak tonen</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="spamCheck">
        <property name="text">
         <string>Spam meldingen binnenhalen</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="spamDeleteCheck">
        <property name="text">
         <string>Spots met veel meldingen verwijderen</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QCheckBox" name="filterManyCatCheck">
        <property name="text">
         <string>Spots met veel categorien weigeren</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Nieuwe spots</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QRadioButton" name="meteenTonenRadio">
        <property name="text">
         <string>Meteen tonen</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QRadioButton" name="naUpdateRadio">
        <property name="text">
         <string>Tonen zodra er op &quot;update&quot; geklikt wordt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QCheckBox" name="updateOnStart">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Bij opstarten automatisch updaten</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QCheckBox" name="filterHCCheck">
        <property name="text">
         <string>Nieuwe spots moeten met SN 1.7+ gespot zijn</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Anti-spam instellingen</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="warmcacheCheck">
        <property name="text">
         <string>Gegevens in cache laden bij opstarten</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="Dutch" country="Netherlands"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
